lib/node_modules/@stdlib/lapack/base/dgttrf/lib/base.js

```diff
@@ -0,0 +1,172 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+
+
+// MAIN //
+
+/**
+* Computes an `LU` factorization of a real tri diagonal matrix `A` using elimination with partial pivoting and row interchanges.
+*
+* ## Notes
+*
+* -   On exit, `DL` is overwritten by the multipliers that define the matrix `L` from the `LU` factorization of `A`.
+* -   On exit, `D` is overwritten by the diagonal elements of the upper triangular matrix `U` from the `LU` factorization of `A`.
+* -   On exit, `DU` is overwritten by the elements of the first super-diagonal of `U`.
+* -   On exit, `DU2` is overwritten by the elements of the second super-diagonal of `U`.
+* -   On exit, for 0 <= i < n, row i of the matrix was interchanged with row IPIV(i).  IPIV(i) will always be either i or i+1; IPIV(i) = i indicates a row interchange was not required.
+*
+* @private
+* @param {NonNegativeInteger} N - order of matrix A.
+* @param {Float64Array} DL - sub diagonal elements of A.
+* @param {integer} sdl - stride length for `DL`
+* @param {NonNegativeInteger} odl - starting index of `DL`
+* @param {Float64Array} D - diagonal elements of A.
```

Comment:
```suggestion
* @param {Float64Array} D - diagonal elements of A
```

Periods here and elsewhere